cypress/support/commands.ts

```diff
@@ -119,3 +119,20 @@ Cypress.Commands.add('loadLandingPage', () => {
 
   cy.resetToDefaultLayout();
 });
+
+Cypress.Commands.add('removeWidget', (widgetId: string) => {
+  // runner on jenkins is a bit slow, wait for a long while
+  const jenkinsWait = 5000;
```

Comment:
Does the `.should('not.exist');` not automatically wait the default timeout before failing the test?